### PR TITLE
Table_Tag_Priya_work

### PR DESCRIPTION
**Pull request by priyaughade**

### CODE REVIEW

**Review by MKadeer17 — APPROVED** [review]

**Review by dnyaneshwarkhule — APPROVED** [review]

File: priya/HTML/PriyaAssign4Q3.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <TItle>
+
```
Comment (Pawan1005) [inline priya/HTML/PriyaAssign4Q3.html]: add title

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
remove extra spaces form files and add title

File: priya/HTML/PriyaAssign4Q3.html (hunk 1 above)
Comment (MKadeer17) [reply priya/HTML/PriyaAssign4Q3.html]: Priya add title tag

Comment (priyaughade) [reply priya/HTML/PriyaAssign4Q3.html]: Already Added

**Review by dnyaneshwarkhule — APPROVED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Pull request merged**